definintion-of-oss-or-iss-project.md

```diff
@@ -0,0 +1,49 @@
+# Title
+
+The Definition of an Open Source Software (OSS) or InnerSource Software (ISS) Project.
+
+## Definition(s)
+
+The following are the components are essential for a project to be declared an Open Source Software
+or InnerSource Software project. Note, the distinction between Open Source and InnerSource is merely
+the network topology surrounding the source control management system instance. If the world at large
+can view the code base, the project Open Source. On the other hand if any portion of the project's source 
+code lives on a restricted network topology such that it is not available to be read by the world at 
+large then we deem it InnerSource. 
+
+We will use the term "population" in the following definitions 
+to reference the user base that has readership based upon the respective network topology. Further, we
+will use the term "owner" to refer to one who has privileges to delete the code base; we will refer to anyone
+who has commits in a repository or documentation or issue list as a "contributor"; we will refer to any person who can 
+push code to the repository a “committer”; we call any person using the repository a “user.” Naturally, the progression 
+of trust follows (most to least trusted) as
+
+1. Owners (commonly referred to as project committee members)
+2. Committers
+3. Contributors
+4. Users
+
+### Tenents of an OSS/ISS Project
```

Comment:
I think you men Tenets

Reply:
I'm good at spellng :-p thanks :-)

Reply:
https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/182/commits/db1cdea89b550380c63f6b856f3315953321436c fixed this, resolving this.